proxy-robots-module/src/main/java/orwell/proxy/Cli.java

```diff
@@ -66,15 +66,15 @@ private void help() {
         logback.warn("Exiting program");
 
         // This prints out some help
-        HelpFormatter formatter = new HelpFormatter();
+        final HelpFormatter formatter = new HelpFormatter();
 
         formatter.printHelp("-f filepath OR -u http://fake.url", options);
         System.exit(0);
     }
 
-    private ConfigFactoryParameters file(String filePath) {
-        File f = new File(filePath);
-        if (!f.exists() || f.isDirectory()) {
+    private ConfigFactoryParameters file(final String filePath) {
+        final File file = new File(filePath);
+        if (!file.exists() || file.isDirectory()) {
             logback.error("File " + filePath + " not found, exiting program");
             System.exit(0);
```

Comment:
Are you sure you want to call exit it seems it would be more appropriate in the main ?